code/stack.yaml

```diff
@@ -69,6 +69,24 @@ extra-deps:
 - multiplate-0.0.3@sha256:5d2ffc50d23c55008100b7a8b70f77b5fcce08c6f23822c5a6adc5b5b9356a32
 - unicode-names-3.2.0.0@sha256:fef7241d93170e26265e84553090253a5e8ee207645d47cf271816f5834d07d2
 - unicode-properties-3.2.0.0@sha256:239766a6ac4322329353f6b9cd546024fd8c5f0235c8e32b3cba2d6bd245a699
+- clay-0.14.0@sha256:a50ba73137a39c55e89f24a7792107ec40ba07320b2c5ff7932049845c50ffc9,2204
+- dir-traverse-0.2.3.0@sha256:adcc128f201ff95131b15ffe41365dc99c50dc3fa3a910f021521dc734013bfa,2137
+- extensions-0.1.0.0@sha256:b8105dc43a57b0b3b54879e8dbb905676dfee3e8b59301fefbf2409a0fe95710,4447
+- tomland-1.3.3.2@sha256:887dc39a8c9819deb8fcb6fde72e87dad4c94108b1736a5bf7215ccf3117bd0f,9474
+- trial-0.0.0.0@sha256:ebd93f3485dd7f0ce8426fa46b500f26edbea285c2890150123e1b0f6a92c7db,4410
+- trial-optparse-applicative-0.0.0.0@sha256:ee878dbf412849d4fb52d7bca93bec45a6392a93e03a46bd320109b9a0ff2a38,2540
+- trial-tomland-0.0.0.0@sha256:c10c5c3f94e3dcfcad4a66cf70682ace77d5381596aff91fa2030636d07de977,2541
+- validation-selective-0.2.0.0@sha256:e1ab5482dede8bf676d729a09109c7c5f798363b9d458e4197a27afb8b48fdd3,3907
+- colourista-0.1.0.2@sha256:87b6c096563c3dc5afb1161f06891235370862fc3406406f1f10f1864e55e0a1,3364
+- slist-0.2.1.0@sha256:679013d3a03ed1f09fc46cc6ef87f94cc36a1320b2bf4fe4f91d102f85277528,3513
```

Comment:
Can you please add a note mentioning that these are here specifically for `stan`?

Reply:
And specifically, tested against GHC 9.4.8.

Reply:
Yes I will add that

Reply:
Do they really all need to be pinned that hard?

Reply:
No, they do not have to be this should also work
- clay>=0.14.0
- dir-traverse>=0.2.3.0
- extensions>=0.1.0.0
- tomland>=1.3.3.2
- trial>=0.0.0.0
- trial-optparse-applicative>=0.0.0.0
- trial-tomland>=0.0.0.0
- validation-selective>=0.2.0.0
- colourista>=0.1.0.2
- slist>=0.2.1.0

Reply:
It's usually a good habit to give upper bounds too, since major version bumps tend to break things.  So
- clay < 1.0.0
- tomland < 2.0.0.0
and so on.

Reply:
Like this
- clay>=0.14.0 && <0.15.0
- dir-traverse>=0.2.3.0 && <0.3.0.0
- extensions>=0.1.0.0 && <0.1.0.2
- tomland>=1.3.3.2 && <1.4.0.0
- trial>=0.0.0.0 && <0.1.0.0
- trial-optparse-applicative>=0.0.0.0 && <0.1.0.0
- trial-tomland>=0.0.0.0 && <0.1.0.0
- validation-selective>=0.2.0.0 && <0.3.0.0
- colourista>=0.1.0.2 && <0.2.0.0
- slist>=0.2.1.0 && <0.3.0.0

Reply:
Well, that might be too tight, but we can certainly try it and see.

Reply:
Actually, they have to be hard-pinned otherwise it doesn't work